Zend/zend_vm_def.h

```diff
@@ -9549,6 +9549,71 @@ ZEND_VM_HANDLER(202, ZEND_CALLABLE_CONVERT, UNUSED, UNUSED)
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(204, ZEND_FRAMELESS_ICALL_0, UNUSED, UNUSED)
+{
+	USE_OPLINE
+	SAVE_OPLINE();
+	zend_frameless_function_0 function = zend_frameless_function_0_list[opline->extended_value];
+	function(EX_VAR(opline->result.var));
+	if (EG(exception)) {
+		ZVAL_NULL(EX_VAR(opline->result.var));
+		HANDLE_EXCEPTION();
+	}
+	ZEND_VM_NEXT_OPCODE();
+}
+
+ZEND_VM_HANDLER(205, ZEND_FRAMELESS_ICALL_1, CONST|TMPVAR|CV, UNUSED)
+{
+	USE_OPLINE
+	SAVE_OPLINE();
+	zend_frameless_function_1 function = zend_frameless_function_1_list[opline->extended_value];
+	function(EX_VAR(opline->result.var), GET_OP1_ZVAL_PTR_DEREF(BP_VAR_R));
+	FREE_OP1();
```

Comment:
Some additional care here is necessary from the handlers. If some argument is mutated it first needs to be copied. I don't think that's usually done, because for arrays we'd already need code that separates it, and other values are usually also just unwrapped.

Maybe it would make sense to document somewhere how frameless handlers work. I'll try to write a few words somewhere.